### ISSUE TITLE
.NET August 2022 Updates - .NET 6.0.8, .NET Core 3.1.28

### ISSUE THREAD

**Issue by dcwhittaker** [dotnet/core#7682]
Release Notes
[6.0.8](https://github.com/dotnet/core/blob/main/release-notes/6.0/6.0.8/6.0.8.md)
[3.1.28](https://github.com/dotnet/core/blob/main/release-notes/3.1/3.1.28/3.1.28.md)

Please report any issues you find either by responding to this issue, creating a new issue or creating a new issue in one of the following repos:

* [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore/issues) - for ASP.NET Core questions
* [dotnet/efcore](https://github.com/dotnet/efcore/issues) - for Entity Framework Core questions
* [dotnet/sdk](https://github.com/dotnet/sdk/issues) - for CLI tools and questions
* [dotnet/runtime](https://github.com/dotnet/runtime/issues) - for runtime, API, or installer issues and questions
* [dotnet/winforms](https://github.com/dotnet/winforms/issues) - for WinForms issues
* [dotnet/wpf](https://github.com/dotnet/wpf/issues) - for WPF issues
* [nuget/home](https://github.com/nuget/home/issues) - for NuGet questions and issues

Distro | 6.0.8 | 3.1.28
-- | -- | -- 
Ubuntu 18.04 |✅ |✅
Ubuntu 20.04 |✅ |✅
Ubuntu 22.04 |✅ |
Centos 7 |✅ |✅
Debian 10 |✅ |✅
Debian 11 |✅ |✅
Fedora 35 |✅ | ✅
Fedora 36|✅ | ✅
OpenSUSE 15|✅ |✅
Oracle|✅ |✅

Note: This list refers to the Microsoft-provisioned feeds (packages.microsoft.com) and does not in any way represent direct availability in distros (eg RHEL, Fedora).

Known Issues

If there are any issues with this release we will track them here and check issues off as they're resolved. See the linked issues for details on progress and resolution details.

**Comment by mockjv** [comment]
Just an FYI, the README.md link for the 6.0.8 and 3.1.28 release notes has an invalid path (still has the old version in the parent directory):

`[6.0.8]: release-notes/6.0/6.0.7/6.0.8.md`
`[3.1.28]: release-notes/3.1/3.1.27/3.1.28.md`

**Comment by dcwhittaker** [comment]
@mockjv - I think it's fixed (unless I'm looking in the wrong place)

**Comment by mockjv** [comment]
@dcwhittaker Unfortunately I think I still see it @ https://github.com/dotnet/core/blob/main/README.md on both `main` and the `6.0.8` tag.

**Comment by dcwhittaker** [comment]
@mockjv _ahh_ I see now. Fixed! Thank you :)

**Comment by andschwa** [comment]
I am running into a type load exception that bubbles up through PowerShell's `TabExpansion2`, and eventually crashes a completion unit test in the PowerShell Extension for VS Code's CI/CD pipeline, which means I'm blocked on releasing a new version of the extension with this runtime until I figure out what's exploded: https://github.com/PowerShell/PowerShellEditorServices/issues/1877

**Comment by MaxG117** [comment]
I ran into a problem with .NET SDK 6.0.303 that doesn't exist in 6.0.300. I think this is the right place to report this problem.

We use a batch file to build our project for various targets. Here's the relevant excerpt:

```
set OUR_VERSION=11.22.33.44
set MSBuildParameters=FileVersion=%OUR_VERSION%;AssemblyVersion=%OUR_VERSIONR%
dotnet build -c Release -f net6.0 /p:%MSBuildParameters% somepath\some.csproj
dotnet build -c Release -f netstandard2.0 /p:%MSBuildParameters% somepath\some.csproj
```

This stopped working with .NET SDK 6.0.303 and produces the following error:

```
Microsoft (R) Build Engine version 17.2.0+41abc5629 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 8/18/2022 12:03:01 PM.
     1>Project "somepath\some.csproj" on node 1 (Restore target(s)).
     1>_GetAllRestoreProjectPathItems:
         Determining projects to restore...
     1>C:\Program Files\dotnet\sdk\6.0.303\NuGet.targets(130,5): error : '11.22.33.44;AssemblyVersion=11.22.33.44' is not a valid version string. (Parameter 'value') [somepath\some.csproj]
     1>Done Building Project "somepath\some.csproj" (Restore target(s)) -- FAILED.
```

Something is either swallowing the "FileVersion=" part of the build parameters, or maybe something else changed. I tried 6.0.400 with the same result. Again, no such problem occurs in 6.0.1x or 6.0.300 SDKs.

**Comment by dcwhittaker** [comment]
Closed in favor of https://github.com/dotnet/core/issues/7791

**Comment by petertirrell** [comment]
Can anyone provide any guidance on CVE-2022-34716 and System.SecurityCryptography.Xml.dll? My understanding was that updating our SDK to 3.1.28 or higher should resolve this.  But it's still showing up on our scans and as far as I can tell our build SDK and runtime should all meeting the fix criteria. The file version of the dll on our build artifacts still reads 11/15/2019, v4.700.19.56404, so I'm not sure how the "fixed" version is supposed to be generated.
